pkg/setting/setting_unified_alerting_test.go

```diff
@@ -11,33 +11,38 @@ import (
 )
 
 func TestCfg_ReadUnifiedAlertingSettings(t *testing.T) {
+	require := require.New(t)
 	cfg := NewCfg()
 	err := cfg.Load(CommandLineArgs{HomePath: "../../", Config: "../../conf/defaults.ini"})
-	require.NoError(t, err)
+	require.NoError(err)
 
+	ua := cfg.UnifiedAlerting
```

Comment:
lmk if you see anything problematic about this, but I find it much more readable.